### ISSUE TITLE
Manual mode

### ISSUE THREAD

**Issue by TimothyPPerrett** [TimothyPPerrett/Smart-Fan#17]
Would be nice to have a mode that lets you nix the matter stuff entirely and run it off the buttons alone, for times when the network isn't there or an automation is doing weird stuff in the middle of the night.

**Comment by TimothyPPerrett** [comment]
A thought: Is this necessary? Worth checking to see if the setup code **has** to be blocking. None of the Matter device code seems to rely on there being a network connection.

Maybe instead of a manual mode I can just remove the `delay` calls in `setup`. Will be able to test for sure once #2 is done.

**Comment by TimothyPPerrett** [comment]
That said, it wouldn't help in the "Matter control is on the fritz" scenario.
